gradle/installer-json.gradle

```diff
@@ -1,7 +1,10 @@
 import groovy.json.JsonOutput
 
 tasks.register("generateInstallerJson", GenerateInstallerJsonTask) {
-	configurations = [ common: "installer" ]
+	configurations = [ common: "installer", development: "development"]
+	repositoryOverrides = [
+	    "mixinextras-fabric": "https://repo.maven.apache.org/maven2/"
+	]
```

Comment:
I know we discussed otherwise, but I think its best to also add this to the launchwrapper json. Even though its not something we use, or test there is no reason not to do it.

I think the other option would be to discuss removing launch wrapper support (as another PR). There was an idea for loader to be able to emulate being LW.